pkg/operator/ceph/object/user/controller.go

```diff
@@ -339,17 +339,22 @@ func generateUserConfig(user *cephv1.CephObjectStoreUser) object.ObjectUser {
 	return userConfig
 }
 
+func generateCephUserSecretName(u *cephv1.CephObjectStoreUser) map[string]string {
+	m := make(map[string]string)
+	m["secretName"] = fmt.Sprintf("rook-ceph-object-user-%s-%s", u.Spec.Store, u.Name)
```

Comment:
Why include this in a map? I'd suggest this method simply return the secret name. Then you can add it to the status in a map where needed.

Reply:
Or use this function to return the `Info` field.

Reply:
To be clear, I'm suggesting this:
```
func generateCephUserSecretName(u *cephv1.CephObjectStoreUser) string {
	return fmt.Sprintf("rook-ceph-object-user-%s-%s", u.Spec.Store, u.Name)
}
```